annotations/src/main/java/org/neo4j/annotations/api/PublicApiAnnotationProcessor.java

```diff
@@ -148,7 +148,7 @@ private void generateSignature() throws IOException
             StringBuilder sb = new StringBuilder();
             for ( final String element : publicElements )
             {
-                sb.append( element ).append( '\n' );
+                sb.append( element ).append( System.lineSeparator() );
```

Comment:
Our build system did not like this change, we compare to the files in `annotations/src/test/resources/signatures` which will have `\n` in our case. I'm not sure how to address that.

Reply:
Well, in this case I added `.gitattributes` to ensure all `PublicApi.txt` are ending with `\n`, hope it is fine now.

I will fully test it later locally, by recloning the repository, since local `.gitattributes` changes are not immediately followed.

Reply:
You only need the git attributes for the files in the `annotations/src/test/resources/signatures` and `annotations/src/test/resources/META-INF/services` directories.